platform-grpc-service-framework/src/main/java/org/hypertrace/core/serviceframework/grpc/GrpcPlatformServiceContainer.java

```diff
@@ -51,6 +48,8 @@ abstract class GrpcPlatformServiceContainer extends PlatformService {
   private final HealthStatusManager healthStatusManager = new HealthStatusManager();
   private InProcessGrpcChannelRegistry grpcChannelRegistry;
   private HealthBlockingStub healthClient;
+  private static final String MAX_LATENCY_HISTOGRAM_BUCKET_MS = "metrics.latency.bucket.max.millis";
+  private static final long DEFAULT_MAX_LATENCY_HISTOGRAM_BUCKET_MS = 10000;
```

Comment:
nit - prefer using durations over millis. More readable, especially in config (e.g. 10000 vs 10s)
```suggestion
  private static final String MAX_LATENCY_HISTOGRAM_BUCKET_DURATION = "metrics.latency.bucket.max.duration";
  private static final long DEFAULT_MAX_LATENCY_HISTOGRAM_BUCKET_DURATION = Duration.ofSeconds(10);
```